projects/storefrontlib/src/shared/components/media/media.component.html

```diff
@@ -5,4 +5,5 @@
   [attr.alt]="media.alt"
   (load)="loadHandler()"
   (error)="errorHandler()"
+  alt="image"
```

Comment:
I'm not sure why we did this actually. I do think we should fix the false positive on sonar, but with this setup, we'd get a `alt="image"` property in the resulting html when the business user hasn't provided a alt message in the content. That's actually not what we want.